bhdr.js

```diff
@@ -2,10 +2,9 @@
 
 const Histogram = require('native-hdr-histogram')
 const histutils = require('hdr-histogram-percentiles-obj')
-const chalk = require('chalk')
-const colors = ['red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white', 'gray']
 const console = require('console') // for proxyquire in tests
 const process = require('process') // for proxyquire in tests
```

Comment:
Good to know. I hope we do not move to ES6 mods in core, but when it happens we will see. Happy to get a PR if you have a better way for dealing with this.